src/emqx_acl_cache.erl

```diff
@@ -27,8 +27,9 @@
         , get_cache_max_size/0
         , get_cache_ttl/0
         , is_enabled/0
-        , put_discarded_stamp/0
-        ]).
+        , start_cache_drain/0
+        , stop_cache_drain/0
+]).
```

Comment:
Small nitpick: the closing square bracket is typically indented at the same level as the previous comma